### PR TITLE
Feature/data integration seller catalogue

### PR DESCRIPTION
**Pull request by marcoalfonso**

### CODE REVIEW

File: apps/orams/components/ResultsTable/ResultsTable.js [hunk 1]
```diff
@@ -12,48 +12,75 @@ class ResultsTable extends Component {
...
-                    {supplier.phone}
-                  </span>
-                  <span className={styles.email}>
-                    <a href={'mailto:' + supplier.email}>
-                      {supplier.email}
-                    </a>
-                  </span>
+          {categories &&
+            categories.map((category, id = uniqueID()) =>
+              <div key={id + category.category}>
```
Comment (matt-sm) [inline apps/orams/components/ResultsTable/ResultsTable.js]: this should be `category.name`

File: apps/orams/components/SellerCatalogue/SellerCatalogue.js [hunk 2]
```diff
@@ -1,85 +1,42 @@
...
-      phone: '1300 762 989',
-      email: 'referral@rehabmanagement.com.au'
-    },
-    {
-      name: 'Rehab Management',
-      price: '$192.50',
-      phone: '1300 762 989',
-      email: 'referrals@rehabmanagement.com.au'
+    if (this.props.errorMessage) {
+      ;<PageAlert as="error">
```
Comment (matt-sm) [inline apps/orams/components/SellerCatalogue/SellerCatalogue.js]: typo `;` ?

File: apps/orams/components/SellerCatalogue/SellerCatalogue.js [hunk 3]
```diff
@@ -1,85 +1,42 @@
...
-      price: '$192.28',
-      phone: '1300 762 989',
-      email: 'referral@rehabmanagement.com.au'
-    },
-    {
-      name: 'Rehab Management',
-      price: '$192.50',
-      phone: '1300 762 989',
-      email: 'referrals@rehabmanagement.com.au'
+    if (this.props.errorMessage) {
```
Comment (matt-sm) [inline apps/orams/components/SellerCatalogue/SellerCatalogue.js]: I could not get this to display - are you checking the right prop?

Comment (marcoalfonso) [reply apps/orams/components/SellerCatalogue/SellerCatalogue.js]: Yes it is, same prop as in marketplace. Can we merge this?

**Pull request merged**